ui/templates/certificate.html

```diff
@@ -147,5 +148,30 @@ <h4 class="signatory-name">{{ signatory.name }}</h4>
     </div>
   </main>
 </div>
+<div class="wrapper-accomplishment-metadata">
+    <div class="accomplishment-metadata">
+      <div class="accomplishment-metadata-bit accomplishment-metadata-aboutplatform">
+        <h3 class="accomplishment-metadata-title">
+          About edX
+        </h3>
+        <p class="accomplishment-metadata-copy">
+          <a href="https://www.edx.org"> edX</a> offers interactive online classes and
+          MOOCs from the world’s best universities, including MIT, Harvard, Berkeley, University of Texas,
+          and many others. edX is a non-profit online initiative created by founding partners Harvard and MIT.
+        </p>
+      </div>
+      <div class="accomplishment-metadata-bit accomplishment-metadata-aboutcert">
+        <h3 class="accomplishment-metadata-title">
+          About the edX MicroMasters credential
+        </h3>
+        <p class="accomplishment-metadata-copy">
+          An edX MicroMasters credential signifies that the learner has completed a group of courses that add up
+          to a rich understanding of an area of study. Completion of MicroMasters Programs requires that learners earn
+          <a href="https://www.edx.org/verified-certificate">Verified Certificates</a>
+          from each of the courses included in the Program.
+      </p>
+    </div>
+  </div>
+</div>
```

Comment:
This isn't included in the printed version at edX, so I'd like to make sure that we don't include it in the printed version either.